cmd/courier/watch.go

```diff
@@ -47,16 +48,21 @@ func ServeMetrics(ctx cx.Context, r driver.Registry) {
 	n.Use(reqlog.NewMiddlewareFromLogger(l, "admin#"+c.SelfPublicURL().String()))
 	n.Use(r.PrometheusManager())
 
-	if tracer := r.Tracer(ctx); tracer.IsLoaded() {
-		n.Use(tracer)
-	}
-
 	n.UseHandler(router)
 
-	server := graceful.WithDefaults(&http.Server{
-		Addr:    c.MetricsListenOn(),
-		Handler: n,
-	})
+	var server *http.Server
```

Comment:
It would probably be easier to have `var h http.Handler` and wrap it with otelx if tracing is enabled, instead of instantiating the server in the if statement:

```
var n http.Handler = negroni.New()

// ...

if tracer.isLoaded() {
  n = otelx.NewHandler...
}

server := ...